support/Pipelines/Bbh/ControlId.py

```diff
@@ -89,6 +91,12 @@ def Residual(
         chi_Kerr_B_x: float,
         chi_Kerr_B_y: float,
         chi_Kerr_B_z: float,
+        center_A_x: float,
+        center_A_y: float,
+        center_A_z: float,
```

Comment:
Question: would it make sense to control the offset in xyz from the Newtonian COM rather than center_A? Seems nicer because the initial guess is 0, but I haven't thought carefully about how this would impact the control loop Jacobian.

Reply:
That's a good idea. I've tested it, and I believe it leads to better results. CoM and Padm still don't converge in some cases, which I think might be related to the low accuracy in their computation. Until this is resolved, I've disabled their control by default.